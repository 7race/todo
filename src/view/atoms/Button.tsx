import styled from 'styled-components';

export const Button = styled.button`
  padding: 5px;
  font-size: 16px;
  font-weight: bold;
  background-color: violet;
  color: ${({ theme }) => theme.colors.white};
  border: 2px;
`;
