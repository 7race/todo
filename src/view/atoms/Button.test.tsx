import { render } from '@tests-utils';
import { Button } from './Button';

describe('button test', () => {
  it('correct styles', () => {
    const { container } = render(<Button />);
    expect(container).toMatchSnapshot();
  });
});
