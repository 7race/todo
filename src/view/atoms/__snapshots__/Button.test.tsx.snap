// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button test correct styles 1`] = `
<div>
  <button
    class="sc-bczRLJ kTUTb"
  />
</div>
`;
